hw_1/Collection - dna converter.py

```diff
@@ -0,0 +1,78 @@
+# dna/rna convert program
+
+# dictionary for complement, transcription reverse transcription
+dict_complement = {'A': 'T', 'T': 'A', 'C': 'G', 'G': 'C',
+                   'a': 't', 't': 'a', 'c': 'g', 'g': 'c'}
+dict_transcript = {'A': 'U', 'T': 'A', 'C': 'G', 'G': 'C',
+                   'a': 'u', 't': 'a', 'c': 'g', 'g': 'c'}
+dict_reverse_trans = {'A': 'T', 'U': 'A', 'C': 'G', 'G': 'C',
+                      'a': 't', 'u': 'a', 'c': 'g', 'g': 'c'}
+
+
+# function for complement, transcription and reverse transcription
+def foo_common(seq, dictionary):
+    output = str()
+    for j in range(len(seq)):
+        output += dictionary[seq[j]]
+    return output
+
+
+# function for reverse sequence
+def foo_reverse(seq):
+    return seq[::-1]
+
+
+# function for reverse complement
+def foo_reverse_complement(seq, dictionary):
+    output = str()
+    for j in range(len(seq)-1, -1, -1):
+        output += dictionary[seq[j]]
```

Comment:
Удобнее последовательно применить функции для комплементирования и реверса. А тут, получается, велосипед
```suggestion
    output = foo_reverse(foo_complement(seq, dictionary))
```